spec/services/nrtm_json_spec.rb

```diff
@@ -54,11 +54,11 @@
       let(:json) { described_class.new(tournament) }
 
       before do
-        r1 = create(:round, stage: cut, completed: true)
+        r1 = create(:round, number: 2, stage: cut, completed: true)
```

Comment:
why are these changes included?

Reply:
I had a problem where this test failed locally because the rounds showed up in the wrong order. The model for round has `default_scope { order(number: :asc) }`, so I think setting the round number should solve the problem. It always gets set in real usage.

Reply:
ah, nice!  I had seen some non-determinism in tests but not that one yet.